setup.go

```diff
@@ -60,12 +62,21 @@ func parseConfig(c *caddy.Controller, config *Config) error {
 		switch value {
 		case "fetch":
 			var url, name string
+			var timeoutInMillis int64 = 10000
+			var err error
 			switch len(args) {
 			case 1:
 				url = args[0]
 			case 2:
 				name = args[0]
 				url = args[1]
+			case 3:
+				name = args[0]
+				url = args[1]
+				timeoutInMillis, err = strconv.ParseInt(args[2], 10, 64)
```

Comment:
I would prefer to support the duration format, see
https://golang.org/pkg/time/#ParseDuration